New Topic: C# with a few initial concepts and terms (#1017)
Codecademy/docs: content/c-sharp/concepts/arrays/terms/length/length.md

---
Title: '.Length'
Description: 'Returns the total number of elements in the array.'
Subjects:
  - 'Computer Science'
  - 'Code Foundations'
Tags:
  - 'Arrays'
CatalogContent:
  - 'learn-c-sharp'
  - 'paths/computer-science'
---

> The **`.Length`** property returns the total number of elements in the array. This includes all dimensions of the array.

Reviewer: ```suggestion
The **`.Length`** property returns the total number of elements in the array, including all dimensions of the array.
```

## Syntax

```pseudo

> int myLegnth = myArray.Length;

Reviewer 2: ```suggestion
int myLength = myArray.Length;
```

```

> `myArray.Length()` returns an `int` that represents the number of elements in `myArray`.

Reviewer: ```suggestion
`myArray.Length` returns an `int` that represents the number of elements in `myArray`.
```

Reviewer: Is it `.Length` or `.Length()`?

## Example

> The following example initializes an array then prints out the number of elements it contains:

Reviewer 2: ```suggestion
The following example initializes an array, then prints out the number of elements it contains:
```
